# Description: Remove WebUIUsers and OpenIDRegistrations from re-indexing,
# as they are not indexed in the first place, which causes a 500 error
# on reindexing.
# This is fixed in upstream bug CHEF-1009 and this patch should be removed
# in a later version (0.8.8+) of the package.
Index: chef-0.8.6/chef-solr/lib/chef/solr.rb
===================================================================
--- chef-0.8.6.orig/chef-solr/lib/chef/solr.rb	2010-03-18 11:03:19.000000000 -0600
+++ chef-0.8.6/chef-solr/lib/chef/solr.rb	2010-03-18 11:03:43.000000000 -0600
@@ -119,7 +119,7 @@
       solr_commit
       
       results = {}
-      [Chef::ApiClient, Chef::Node, Chef::OpenIDRegistration, Chef::Role, Chef::WebUIUser].each do |klass|
+      [Chef::ApiClient, Chef::Node, Chef::Role].each do |klass|
         results[klass.name] = reindex_all(klass) ? "success" : "failed"
       end
       databags = Chef::DataBag.cdb_list(true)
